packages/p2p/src/pages/buy-sell/buy-sell-form.jsx

```diff
@@ -143,6 +161,25 @@ const BuySellForm = props => {
         return max_order_amount_limit_display;
     };
 
+    React.useEffect(() => {
+        buy_sell_store.form_props.setIsSubmitDisabled(
+            !isValid ||
+                isSubmitting ||
+                (buy_sell_store.is_sell_advert && payment_method_names && selected_methods.length < 1)
+        );
+    }, [
+        isValid,
+        isSubmitting,
+        selected_methods.length,
+        buy_sell_store.form_props,
+        buy_sell_store.is_sell_advert,
+        payment_method_names,
+    ]);
+
+    React.useEffect(() => {
+        buy_sell_store.form_props.setSubmitForm(submitForm);
+    }, [submitForm, buy_sell_store.form_props]);
```

Comment:
do we still need this?

Reply:
yeah we still need to set the form props for other props from buy-sell-modal like onCancel, onConfirmClick, setIsSubmitDisabled

Reply:
Didn't change how the props were set and passed to the store, but in the future, we need to change this behaviour, but for now, this still works and has no console errors